manifests/b/BandicamCompany/Bandicam/5.2.0.1855/BandicamCompany.Bandicam.installer.yaml

```
@@ -0,0 +1,20 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: BandicamCompany.Bandicam
+PackageVersion: 5.2.0.1855
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://dl.bandicam.com/bdcamsetup.exe
```

I would suggest using the direct link to the executable
```suggestion
  InstallerUrl: https://dl.bandicam.com/old/BDCAMSETUP_ENG_5_2_0_1855.EXE
```

---

Thanks!